cmd/main.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/okteto/test/pkg/cert"
+	"github.com/okteto/test/pkg/command"
+	"github.com/okteto/test/pkg/input"
+	"github.com/spf13/afero"
+)
+
+func main() {
+	args := os.Args[1:]
+	environ := os.Environ()
+	envVars := getEnvVars(environ)
+	fs := afero.NewOsFs()
+
+	userInput, err := input.NewInput(args, envVars)
+	if err != nil {
+		logger := slog.New(&slog.TextHandler{})
+		logger.Error("Error parsing input", slog.String("error", err.Error()))
+		os.Exit(1)
+	}
+
+	logLevel := parseLogLevel(userInput.LogLevel)
```

Comment:
We should consider also the environment variable `RUNNER_DEBUG`. The old script was taking it into account and it is the environment variable added by GitHub when debug logs are enabled in GitHub

Reply:
I removed when removing the logger wrapper but I've added it again